### ISSUE TITLE
预训练时loss为nan

### ISSUE THREAD

**Issue by iDecisionTree** [jingyaogong/minimind#450]
<img width="855" height="1208" alt="Image" src="https://github.com/user-attachments/assets/ffd743b1-bd7c-4561-91b1-5300f3a77eea" />

<img width="485" height="363" alt="Image" src="https://github.com/user-attachments/assets/f5381be8-a8c6-4468-a267-e9d64eace531" />

**Comment by dao86** [comment]
loss.item() * args.accumulation_steps,  和日志直接相关的代码，你需要先确认一下这几个变量，然后逐步排查以下几个问题，
标签越界 | loss = nan | 检查 label 范围
输入异常 | loss = nan | 检查 inputs 中是否有 nan/inf
梯度爆炸 | loss 突然变 nan | 梯度裁剪、BatchNorm
混合精度问题 | loss = nan | 暂时关闭 AMP 测试

**Comment by jingyaogong** [comment]
lr ⬇️
